src/firebase-sn.ts

```diff
@@ -1,4 +1,4 @@
-import * as Firebase from 'firebase'
+import {firebase} from './process-firebase'
 import {makeCollections} from './collections'
```

Comment:
tbh i think this is an unnecessary abstraction at this point, ive been poking around in a branch with this, and i am finding a clean approach of:

* seneca-firebase-model: doesnt care what "model name" you push it, its just using it as the path for the records youre dealing with, this is talking directly to the firebase reference that it was passed in its options
* individual model plugins, eg Profiles
* the rest of the app only uses the model plugins

So the only place you're explicitly defining the firebase path of a given model is in its plugin.  We dont have to worry about crap like writing to "Opp" rather than "Opps", because we're sending a message to the Opps plugin, and it wont land if we send it to the "Opp" plugin.